### ISSUE TITLE
ULS UNITS ASIA - Help link hover needs to meet ratio

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#455]
"Help for International Students" link at the top right of the homepage on hover fails 4.5:1
